src/pages/PermissionRequest.vue

```diff
@@ -156,6 +202,19 @@ async function initFromChainId(chainId: string) {
     chainInit = true
     const chainDetails = await getChainDetails(appId, chainId)
     chainConfig.value = { ...chainDetails }
+    const rpcConfig = {
+      rpcUrls: chainDetails.rpc_url,
+      chainId: chainDetails.chain_id,
+      chainName: chainDetails.chain_name,
+      blockExplorerUrls: chainDetails.rpc_url,
+      nativeCurrency: {
+        symbol: chainDetails.currency,
+        decimals: 18,
+      },
+      favicon: '',
```

Comment:
Why is this blank?

Reply:
icon is not part of the response and it is not used anywhere